packages/manager/src/features/Linodes/LinodesDetail/LinodeConfigs/LinodeConfigDialog.tsx

```diff
@@ -667,12 +659,9 @@ export const LinodeConfigDialog = (props: Props) => {
 
   const networkInterfacesHelperText = (
     <Typography>
-      Configure the network that a selected interface will connect to (
-      {vpcEnabled
-        ? '"Public Internet", VLAN, or VPC'
-        : 'either "Public Internet" or a VLAN'}
-      ) . Each Linode can have up to three Network Interfaces. For more
-      information, see our{' '}
+      Configure the network that a selected interface will connect to
+      &quot;Public Internet&quot;, VLAN, or VPC. Each Linode can have up to
```

Comment:
Do we need the `&quot;`?
```suggestion
      "Public Internet", VLAN, or VPC. Each Linode can have up to
```

Reply:
ESLint recommended escaping them

![Screenshot 2024-02-12 at 12 34 48 PM](https://github.com/linode/manager/assets/114682940/88a6a18e-08c0-43e1-a608-a3b471d7841b)